tox.ini

```diff
@@ -15,18 +15,28 @@ commands =
     pip install -e .
     py.test {posargs:pymemcache/test/ -m integration}
 
-[testenv:py27-flake8]
+[testenv:py2.7-flake8]
 commands =
     pip install flake8
-    flake8 pymemcache/
+    flake8 --max-complexity=18 pymemcache/
     python setup.py check --restructuredtext
 
-[testenv:py39-flake8]
+[testenv:py3.9-flake8]
 commands =
     pip install flake8
-    flake8 pymemcache/
+    flake8 --max-complexity=18 pymemcache/
     python setup.py check --restructuredtext
 
+[testenv:py2.7-codespell]
```

Comment:
Do we want `codespell` as a dedicated target or do we want it as a sub command of the `flake8` target?

It's up to you to decide. I'm ok with the both versions.